pkg/cluster/ready.go

```diff
@@ -47,6 +47,10 @@ var deploymentWhitelist = []struct {
 		Name:      "cluster-monitoring-operator",
 		Namespace: "openshift-monitoring",
 	},
+	{
+		Name:      "metrics-bridge",
```

Comment:
in the bad old days we didn't wait for prometheus to come up before declaring a cluster ready - but we checked that it did in all the e2es.  Can't we do the same for metrics-bridge and cluster-monitoring-operator?  A bit of a cheat, but it might help our startup times?

Reply:
https://github.com/openshift/openshift-azure/issues/842